movR-ui/my-app/src/Pages/ContactUsPage/ContactUs.css

```diff
@@ -0,0 +1,44 @@
+body {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+      "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+      sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    background-color: #EEE;
+    background-image: url("../../components/registration-component/bgs.jpeg");
+    background-size: cover;
+  }
```

Comment:
this can't be added as `body` tag, you should give it a class, it will mess up with every page otherwise